pkg/controller/configmap/configmap_controller.go

```diff
@@ -156,6 +184,7 @@ func (r *ReconcileConfigMap) Reconcile(request reconcile.Request) (reconcile.Res
 	} else if request.Namespace == "" && request.Name == operatortypes.NetworkCRDName {
 		reqLogger.Info("Reconciling cluster Network CRD change")
 	} else {
+		log.Info(fmt.Sprintf("Received change from unexpected namespace: %s", request.Namespace))
```

Comment:
In which case operator will print this log?

Reply:
This is still a defensive log info. This log will be printed when there is unexpected event change from namespace other than "" or nsx-ncp-operator.  It should have no more such unexpected changes coming since we have added namespace filter for the controller. So the log info maybe incorrect, shall we put this as a debug level log?